src/index.ts

```diff
@@ -276,6 +276,21 @@ async function launchMetals(
       });
     });
 
+    registerCommand("metals.analyze-stacktrace", async () => {
```

Comment:
- [ ] Use this when merged: https://github.com/scalameta/metals-languageclient/pull/184

Reply:
merged

Reply:
👍 I'll change when released.

Reply:
Released and updated.